src/course-unit/CourseUnit.test.jsx

```diff
@@ -146,4 +158,57 @@ describe('<CourseUnit />', () => {
     expect(titleEditField).not.toBeInTheDocument();
     expect(await findByText(newDisplayName)).toBeInTheDocument();
   });
+
+  it('doesn\'t handle creating xblock and displays an error message', async () => {
```

Comment:
```suggestion
  it('doesn\'t handle creating video xblock and displays an error message', async () => {
```

Reply:
Maybe this is was made to be generic intentionally.  In which case, ignore the comment.

Reply:
Yes, you are right. It might be as a generic and can be reverted in next MR.